docs/sources/operators-guide/mimir-runbooks/_index.md

```diff
@@ -70,32 +70,35 @@ How to **fix** it:
 2. **Check if shuffle-sharding shard size is correct**<br />
 
 - When shuffle-sharding is enabled, we target up to 100K series / tenant / ingester assuming tenants on average use 50% of their max series limit.
-- Run the following **instant query** to find tenants that might cause higher pressure on some ingesters:
+- Run the following **instant query** to find tenants that might cause higher pressure on ingesters. The query excludes tenants which are already sharded across all tenants:
 
   ```
-  (
-    sum by(user) (cortex_ingester_memory_series_created_total{namespace="<namespace>"}
-    -
-    cortex_ingester_memory_series_removed_total{namespace="<namespace>"})
+  topk by (pod) (
+    5, # top 5 users per ingester
+    (
+      sum by (user, pod) (
+        cortex_ingester_memory_series_created_total{namespace="<namespace>"} - cortex_ingester_memory_series_removed_total{namespace="<namespace>"}
+      )
+      > on (user) group_left() # show tenants that are exeeding 50% of their series limits added acorss ingesters
+      (
+        max by(user) (cortex_limits_overrides{namespace="<namespace>",limit_name="max_global_series_per_user"})
+        *
+        scalar(max(cortex_distributor_replication_factor{namespace="<namespace>"}))
+        *
+        0.5
+      )
+      > 200000
```

Comment:
where does the `200k` come from? is the idea to make sure that only tenants that have a minimum size get shown?

Reply:
I took the 200k from the previous query, I think your understanding is correct